psamm/datasource/sbml.py

```diff
@@ -711,11 +712,14 @@ def write_model(self, file, model):
         reaction_genes = {}
         reaction_equations = {}
         reaction_names = {}
+        biomass_id = ''
```

Comment:
What if biomass is not set in the YAML model? Maybe this should be `None` by default?